KubeArmor/enforcer/appArmorEnforcer.go

```diff
@@ -1,6 +1,7 @@
 // SPDX-License-Identifier: Apache-2.0
 // Copyright 2021 Authors of KubeArmor
 
+// Package enforcer declaration
```

Comment:
```suggestion
// Package enforcer is responsible for setting up and handling policy updates for supported enforcers including AppArmor, SELinux and BPFLSM
```

Let's move this comment to `enforcer/runtimeEnforcer.go`